Tasks/AzureRmWebAppDeploymentV4/Tests/L0.ts

```diff
@@ -236,7 +236,7 @@ describe('AzureRmWebAppDeployment Suite', function() {
             assert(tr.stdOutContained('set AppServiceApplicationUrl=http://mytestapp.azurewebsites.net'), 'Should have printed: set AppServiceApplicationUrl=http://mytestapp.azurewebsites.net');
             assert(tr.stdOutContained('Active DeploymentId :MOCK_DEPLOYMENT_ID'), 'Should have printed: Active DeploymentId :MOCK_DEPLOYMENT_ID.');
             assert(tr.stdOutContained('DeployWebAppStep for run from zip steps with zip package succeeded'), 'Should have printed: DeployWebAppStep for run from zip steps with zip package succeeded.')
-            assert(tr.stdOutContained('loc_mock_UpdatingAppServiceApplicationSettings {"WEBSITE_RUN_FROM_ZIP":"1"}'), 'Should have printed: loc_mock_UpdatingAppServiceApplicationSettings {"WEBSITE_RUN_FROM_ZIP":"1"}');
+            assert(tr.stdOutContained('loc_mock_UpdatingAppServiceApplicationSettings {"WEBSITE_RUN_FROM_PACKAGE":"1"}'), 'Should have printed: loc_mock_UpdatingAppServiceApplicationSettings {"WEBSITE_RUN_FROM_PACKAGE":"1"}');
```

Comment:
We are using both WEBSITE_RUN_FROM_PACKAGE and WEBSITE_RUN_FROM_ZIP in task . Can you add specific L0 which will give coverage for both ?

Reply:
We dont need WEBSITE_RUN_FROM_ZIP. This is just cleanup.